event-sauce/src/lib.rs

```diff
@@ -235,6 +309,28 @@ where
     }
 }
 
+/// A wrapper trait around [`AggregateConflict`] to handle event-sauce integration boilerplate
+pub trait ConflictEntityBuilder<EDA, EDC>: AggregateConflict<EDA, EDC> + Entity
+where
+    EDA: EventData,
+    EDC: EventData,
+{
+    /// Conflict the entity with an event
+    fn try_conflict<B>(
```

Comment:
Wouldn't this make more sense as `try_flag_conflict` or `try_store_conflict` or something? We're not attempting to _cause_ a conflict, but store its result here, right?

Reply:
I see what you had in mind, when speaking about having to think about the naming - both of those two suggestions are bit weird names, but it is indeed challenging to come up with better ones (I do not have any better ideas). So `try_flag_conflict` it is.